Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -202,6 +204,109 @@ - (BOOL)containsPasteboardTypes:(NSArray*)pasteboardTypes {
 
 /**
  @Status Caveat
+ @Notes only support one item due to platform limit.
+ */
+- (NSArray<NSDictionary<NSString*, id>*>*)items {
+    WADDataPackageView* dataPackageView = _getClipboardContent();
+
+    int supportedTypes = 0;
+    for (NSString* format in dataPackageView.availableFormats) {
+        if ([format isEqualToString:[WADStandardDataFormats text]]) {
+            supportedTypes++;
+        } else if ([format isEqualToString:[WADStandardDataFormats bitmap]]) {
+            supportedTypes++;
+        } else if ([format isEqualToString:[WADStandardDataFormats uri]]) {
+            supportedTypes++;
+        }
+    }
+
+    // at most items will contain item, if there is no item, still need return empty array
+    NSMutableArray<NSDictionary<NSString*, id>*>* items = [NSMutableArray arrayWithCapacity:1];
+
+    NSMutableDictionary* ret = (supportedTypes > 0) ? [NSMutableDictionary dictionaryWithCapacity:supportedTypes] : nil;
+    if (ret) {
```

Comment:
nit: would be much cleaner to just return items early here if supported types == 0